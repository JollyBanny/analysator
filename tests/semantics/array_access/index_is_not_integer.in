var arr1: array [1..100] of integer;

begin
    write(arr1[1 + 999 / 2 - 450]);
end.