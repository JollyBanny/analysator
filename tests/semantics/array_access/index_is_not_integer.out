[Line: 4; Column: 28]	index is not integer
